www-cronrobot/Gemfile

```diff
@@ -64,7 +64,7 @@ group :development do
 end
 
```

Comment:
center the discord image, and it should probably be smaller, it's pretty huge it seems

![image](https://user-images.githubusercontent.com/8007722/132593077-a02d5cb0-fdbd-4297-9dd7-17f6767ec35b.png)